Makefile

```diff
@@ -26,9 +30,23 @@ check-license:
 
 tools: $(TOOLS)
 
+man: $(MAN)
+
+all: $(TOOLS) $(MAN)
+
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+$(MAN): %.1: cmd/*/*.1.md
+	go-md2man -in "$<" -out "$@"
+
+install:
+	install -m 755 $(TOOLS) /usr/local/bin/
+	install -m 644 $(MAN) /usr/local/share/man/man1
+
+uninstall: clean
+	rm -f $(patsubst %,/usr/local/share/man/man1/%,$(MAN)) $(patsubst %,/usr/local/bin/%,$(TOOLS))
```

Comment:
This would read more easily as a substitution reference (with the line I suggested [here][1]).

[1]: https://github.com/opencontainers/image-tools/pull/92#discussion_r90597101